packages/docusaurus-plugin-content-blog/src/index.ts

```diff
@@ -448,11 +448,9 @@ export default function pluginContentBlog(
             `${feedType}.xml`,
           );
           const feedContent = feedType === 'rss' ? feed.rss2() : feed.atom1();
-          try {
-            fs.writeFileSync(feedPath, feedContent);
-          } catch (err) {
+          return fs.outputFile(feedPath, feedContent).catch((err) => {
```

Comment:
Hi

What about using async/await here too. I think we should progressively update to async/await everywhere for consistency

Reply:
Hi @slorber ,
I know about that. But this a `Promise.all`, and should return promise. I dont think `async/await` is suitable. 

`async/await` is good syntax but not fit any case

If you still persist in it, i will update it.

Thanks for you review

Reply:
What is the problem with Promise.all and async/await?

We already have this in other places already

Reply:
like in packages/docusaurus-plugin-content-docs/src/index.ts:464

Reply:
Well, just think this too redundancy —— its unnecessary to write `async/await`.

But if that is a project rule, i will comply with it.